app/main.py

```diff
@@ -1,7 +1,20 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+
+def get_weather(key: str, city: str) -> str:
+    url = "http://api.weatherapi.com/v1/current.json"
+    data = requests.get(f"{url}?key={key}&q={city}")
```

Comment:
Make sure you received a successful response with status code 200 before retrieving the data.